text/reporting_assurance.tex

```diff
@@ -124,10 +124,10 @@ \subsubsection{The Assurances Extrinsic}
   &\mathsf{X}_A \equiv \token{\$jam\_available}
 \end{align}
 
-A bit may only be set if the corresponding core has a report pending availability on it which has not timed out:
+A bit may only be set if the corresponding core has a report pending availability on it:
 \begin{align}
   &\forall a \in \xtassurances, c \in \N_\mathsf{C} :\\
-  &\quad a_f[c] \Rightarrow \rho^\dagger[c] \ne \none \wedge \mathbf{H}_t \le \rho^\dagger[c]_t + \mathsf{U}
+  &\quad a_f[c] \Rightarrow \rho^\dagger[c] \ne \none
```

Comment:
Perhaps we can eliminate the second condition (timeout) from 11.30. 
By this stage, it should already have been reset to the empty set as per 11.18 ($\mathbf{H}_t \ge \rho^\dagger[c]_t + \mathsf{U}$).

![screenshot-2024-12-08-11-15-55](https://github.com/user-attachments/assets/4cf525b1-aef8-42d3-aa82-33763c6a6811)